code/src/Components/Details.css

```diff
@@ -0,0 +1,111 @@
+ .movie-overview {
+    color:white;
+  }
+
+.movie-title {
+    color:white;
+  }
+
+.background {
+    height: 100vh;
+    width: 100vw;
+    position: absolute;
+    overflow: auto;
+  }
+
+.details-container {
+    margin-top: 10vh;
+    margin-left: 5vw;
+  }
+
+ .all-details {
+  position: relative;
+  height: 100vh; /* Set the height of the container to fill the viewport */
+}
+
+
+.back-btn {
+  position: absolute;
+  top: 10px; /* Adjust the top position as needed */
+  left: 10px; /* Adjust the left position as needed */
+  z-index: 1; /* Set a higher z-index to ensure the button is on top of the background */
+  /* Add other styles for the button */
+  margin-left: 5vw;
+}
+
+.details-container img {
+  border: solid 3px white;
+}
+
+button {
+  background-color: #031244; 
+  border: none; 
+  color: white; 
+  padding: 15px 32px; 
+  text-align: center; 
+  text-decoration: none;
+  display: inline-block;
+  font-size: 20px;
+  margin: 4px 2px;
+  cursor: pointer;
+  border-radius: 20px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  opacity: 0.8;
+}
+button:hover {
+  background-color: #040958;
+  color: white;
+  transition: .5 ease;
+  opacity: 1;
+}
+
+button:focus {
+  outline: none;
+  box-shadow: 0 0 0 3px rgba(0,0,0,0.2);
+}
+
+button:active {
+  background-color: #e7e7e7;
+  color: black;
+}
+
+@media screen and (max-width: 800px) {
```

Comment:
The code makes use of media queries to adjust the layout and styling of the page based on the screen size. This is a good approach to ensure that the page is usable and readable on different devices.